src/main/kotlin/racing/domain/RacingCars.kt

```diff
@@ -1,14 +1,16 @@
 package racing.domain
 
-internal class RacingCars(val cars: List<Car>, private val movable: Movable) {
+internal class RacingCars(val cars: List<Car>) {
 
     val winners: List<Car>
         get() {
-            val maxCar = this.cars.maxBy(Car::position)
-            return this.cars.filter { it.position == maxCar!!.position }
+            val maxCar: Car = this.cars.maxWith(compareBy { it.position })
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
            val maxCar: Car = this.cars.maxBy { it.position }
```